libs/ml/include/ml/charge_estimation/types.hpp

```diff
@@ -0,0 +1,28 @@
+#pragma once
+//------------------------------------------------------------------------------
+//
+//   Copyright 2018-2020 Fetch.AI Limited
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+//
+//------------------------------------------------------------------------------
+
+#include <cstdint>
+
+namespace fetch {
+namespace ml {
+
+using MLChargeAmount = std::uint64_t;
```

Comment:
ideally, the naming would indicate that this is in fact not a charge amount but an estimate of the number of operations.

Reply:
renamed to OperationsCount